extra/grafana/README.md

```diff
@@ -14,7 +12,21 @@ This exporter is provided under Best Efforts support by the Pure Portfolio Solut
 5. Check out the features and default values set in the [Pure Storage FlashArray Overview Grafana Dashboard](#Pure-Storage-FlashArray-Overview-Grafana-Dashboard)
 
 # Overview
-So you would like to collect metrics from your Pure Storage FlashArray and monitor your environment on-premise? Well let's walk through it.
+Take holistic overview of your Pure Storage FlashArray estate on-preimise with Prometheus and Grafana to summerise statistics such as:
+  * Array Utilization
+  * Purity OS version
+  * Data Reduction Rate
+  * Number and type of open alerts
+
+Drill down into specific arrays and identify top busy hosts while correlating read and write operations and throughput to quickly highlight or elimiate investigation equiries.
+<br>
+<img src="./images/grafana_purefa_overview_dash_1.png" width="66%" height="66%">
+<img src="./images/grafana_purefa_overview_dash_2.png" width="33%" height="33%">
+<br>
+
+These dashboards are provide an overview of your fleet to give you early indications of potential issues and a look back in time of recent history. Once you are pulling the metrics, you can create your own dashboards bespooke to your environment, even correlating metrics from other technologies. 
```

Comment:
remove 'are'.
...in time to recent...
bespoke, not bespooke

Reply:
Thanks for catching these @sdodsley - corrections made in commit#3